pkg/util/docker/event_pull.go

```diff
@@ -78,10 +78,22 @@ func (d *DockerUtil) processContainerEvent(ctx context.Context, msg events.Messa
 	}
 
 	action := msg.Action
-
-	// Fix the "exec_start: /bin/sh -c true" case
-	if strings.Contains(action, ":") {
-		action = strings.SplitN(action, ":", 2)[0]
+	// Some actions are prefixed then followed `: <string>`.
+	// Extract the prefix from the action here
+	// Example: "exec_start: /bin/sh -c true" case
+	for _, v := range []events.Action{
```

Comment:
Not sure how often `processContainerEvent` is called, but I'm wondering if we should create a private constant array with the events we support.

WDYT?